minifi_main/MiNiFiMain.cpp

```diff
@@ -258,7 +259,11 @@ int main(int argc, char **argv) {
 
     minifi::core::extension::ExtensionManager::get().initialize(configure);
 
-    if (argc >= 3 && std::string("docs") == argv[1]) {
+    if (argc >= 2 && std::string("docs") == argv[1]) {
+      if (argc == 2) {
```

Comment:
would it make sense to also check for too many arguments? e.g. if the user accidentally added a space in one of the paths (or if it already contains spaces but is not properly escaped)

Reply:
yes, good idea, done in 4388f346770f907b725052893d8063d33ab9a6e3